core/src/com/unciv/models/ruleset/unit/BaseUnit.kt

```diff
@@ -133,8 +134,10 @@ class BaseUnit : RulesetObject(), INonPerpetualConstruction {
             textList += FormattedLine(replacementTextForUniques)
         } else if (uniques.isNotEmpty()) {
             textList += FormattedLine()
-            for (uniqueObject in uniqueObjects.sortedBy { it.text })
-                textList += FormattedLine(uniqueObject)
+            uniqueObjects.sortedBy { it.text }.forEach {
```

Comment:
Makes you think - why in two handfuls of repeated 'pedia code displaying uniques is this the only repetition that sorts, and then by the english version? Most are intentionally in json order, so json has some control over the looks. I really can't remember why. Were these uniques long ago a set thus not having stable order??? Might warrant a little test / code merge into some superclass method..